assets/js/components/user-input/UserInputQuestionnaire.js

```diff
@@ -125,6 +126,15 @@ export default function UserInputQuestionnaire() {
 		setActiveSlug( steps[ steps.length - 1 ] );
 	}, [ activeSlugIndex ] );
 
+	useEffect( () => {
+		if ( firstRender ) {
+			setFirstRender( false );
+			return;
+		}
+
+		global.document.getElementById( `googlesitekit-user-input-question-${ activeSlugIndex + 1 }` )?.scrollIntoView( { behavior: 'smooth' } );
```

Comment:
Shouldn't this be simply `activeSlugIndex` rather than +1 since the effect is already triggered by this state changing?

Reply:
Ah, the indexes are mismatched in their base; `activeSlugIndex` references a zero-based index:

<img width="1234" alt="Screenshot 2021-03-18 at 17 15 09" src="https://user-images.githubusercontent.com/90871/111668548-c66e4580-880d-11eb-80ee-6b8ee798172b.png">

but the question IDs start at "1":
<img width="376" alt="Screenshot 2021-03-18 at 17 12 25" src="https://user-images.githubusercontent.com/90871/111668542-c4a48200-880d-11eb-8df6-3bb95d24402a.png">

So this is expected but you're right: it's super-unclear. I've added a comment to clarify it 🙂